docs/examples/ratify-verify-azure-cmd.md

```diff
@@ -1,296 +1,347 @@
     ```
 
-3. Discover the signatures
+## View the signed images associated with signatures
 
-    ```bash
-    # Discover full graph of supply chain content
-    ratify discover -s $IMAGE_DIGEST_REF
-    ```
+Signed images can be viewed with the `notation list` command
 
-4. Verify the full graph of supply chain content
+```bash
+notation list $IMAGE
+```
+## Create and configure Azure Workload Identity
 
-    ```bash
-    # Verify full graph
-    ratify verify -s $IMAGE_DIGEST_REF
-    ```
+Ratify pulls artifacts from a private Azure Container Registry using Workload Federated Identity in an Azure Kubernetes Service cluster. For an overview on how workload identity operates in Azure, refer to the [documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation). You can use workload identity federation to configure an Azure AD app registration or user-assgined managed identity. The following workflow include Workload Identity configuration.
+
+### Set up Workload Identity with AKS, ACR and AKV:
+The official steps for setting up Workload Identity on AKS can be found here.
+
+1. Configure environment variables.
+
+```
+export IDENTITY_NAME=<Identity Name>
+export GROUP_NAME=<Azure Resource Group Name>
+export SUBSCRIPTION_ID=<Azure Subscription ID>
+export TENANT_ID=<Azure Tenant ID>
+export ACR_NAME=<Azure Container Registry Name>
+export AKS_NAME=<Azure Kubernetes Service Name>
+export KEYVAULT_NAME=<Azure Key Vault Name>
+export RATIFY_NAMESPACE=<Namespace where Ratify deployed, defaults to "gatekeeper-system">
+export CERT_DIR=<Directory storing TLS certs>
+export NOTARY_PERM_NAME=<Name of cerficicate file uploaded to Key Vault>
+```
+2. Create a Workload Federated Identity.
+
+```
+az identity create --name "${IDENTITY_NAME}" --resource-group "${GROUP_NAME}" --location "${LOCATION}" --subscription "${SUBSCRIPTION_ID}"
+
+export IDENTITY_OBJECT_ID="$(az identity show --name "${IDENTITY_NAME}" --resource-group "${GROUP_NAME}" --query 'principalId' -otsv)"
+export IDENTITY_CLIENT_ID=$(az identity show --name ${IDENTITY_NAME} --resource-group ${GROUP_NAME} --query 'clientId' -o tsv)
+```
+
+3. Create ACR and enable `AcrPull` role to the workload identity.
+
+```
+az acr create --name "${ACR_NAME}" \
+  --resource-group "${GROUP_NAME}" \
+  --sku Standard
+```
+
+4. Configure user-assigend managed identity
+
+```
+az role assignment create \
+  --assignee-object-id ${IDENTITY_OBJECT_ID} \
+  --role acrpull \
+  --scope subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${GROUP_NAME}/providers/Microsoft.ContainerRegistry/registries/${ACR_NAME}
+```
+
+5. Create OIDC enabled AKS cluster by following the steps below:
+
+```
+# Install the aks-preview extension
+az extension add --name aks-preview
+
+# Register the 'EnableWorkloadIdentityPreview' feature flag
+az feature register --namespace "Microsoft.ContainerService" --name "EnableWorkloadIdentityPreview"
```

Comment:
It might be worth mentioning this step can take ~10 minutes to complete and registration status can be checked by running

```bash
az feature show --namespace "Microsoft.ContainerService" --name "EnableWorkloadIdentityPreview" -o table
```

When it's been successfully registered, the output of the above command should be:

```bash
Name                                                      RegistrationState
--------------------------------------------------------  -------------------
Microsoft.ContainerService/EnableWorkloadIdentityPreview  Registered
```